packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -145,12 +145,12 @@ export default class PciStoragesContainersService {
       );
   }
 
-  getContainer(projectId, containerId, isHighPerfStorage = false) {
+  getContainer(projectId, containerId, region, isHighPerfStorage = false) {
     let promise = null;
     if (isHighPerfStorage) {
       promise = this.$http
         .get(
-          `/cloud/project/${projectId}/region/${OPENIO_DEFAULT_REGION}/storage/${containerId}`,
+          `/cloud/project/${projectId}/region/${region}/storage/${containerId}`,
```

Comment:
Could `region` be null? If so we could fallback to `OPENIO_DEFAULT_REGION`

Reply:
I believe, region won't be null but its always good to check and fallback.